docs/plugins.md

```diff
@@ -390,3 +390,50 @@ Leaflet keeps it simple. If you can think of a feature that is not required by a
 There are no hard requirements on how to create your own plugin, but all developers are encouraged to read the recommendations in the [plugin guide](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md).
 
 Once your plugin is ready, you can submit it: just send a pull request with a new plugin file in [/docs/_plugins/](https://github.com/Leaflet/Leaflet/tree/main/docs/_plugins)to our GitHub repository.
+
+<script>
+function loadRepoData() {
+	const rows = document.querySelectorAll('.plugins table tr');
+	rows.forEach((row) => {
+		try {
+			const repoData = row.querySelector('.repo-data');
+			if (repoData) {
+				const link = row.querySelector('.plugin-repo-url').href;
+				let badges = [];
+				if (link.indexOf('github.com') > -1) {
+					const repo = link.split('github.com/')[1];
+					badges = [
+						`https://badgen.net/github/stars/${repo}`,
+						`https://badgen.net/github/last-commit/${repo}`
+					];
+				}
+				if (link.indexOf('github.io') > -1) {
+					const linkData = link.split('.github.io/');
+					const user = linkData[0].split('//')[1];
+					const repo = `${user}/${linkData[1]}`;
+					badges = [
+						`https://badgen.net/github/stars/${repo}`,
+						`https://badgen.net/github/last-commit/${repo}`
+					];
+				}
+
+				if (link.indexOf('gitlab.com') > -1) {
+					const repo = link.split('gitlab.com/')[1];
+					badges = [
+						`https://badgen.net/gitlab/stars/${repo}`,
+						`https://badgen.net/gitlab/last-commit/${repo}`
+					];
+				}
+
+				const html = '';
+				badges.forEach((badge) => {
+					repoData.innerHTML += `<img src="${badge}" alt=""/><br>`;
```

Comment:
Wouldn't this look better without that `<br/>`, a bit more compact with both badges in the same line?. I'd even try throwing a `white-space: no-wrap` CSS rule to ensure one line.